src/EnergyPlus/GroundTemperatureModeling/FiniteDifferenceGroundTemperatureModel.hh

```diff
@@ -156,11 +155,11 @@ public:
 
     void doStartOfTimeStepInits();
 
-    Real64 getGroundTemp();
+    Real64 getGroundTemp() override;
 
-    Real64 getGroundTempAtTimeInSeconds(Real64 const depth, Real64 const timeInSecondsOfSim);
+    Real64 getGroundTempAtTimeInSeconds(Real64 const depth, Real64 const timeInSecondsOfSim) override;
 
-    Real64 getGroundTempAtTimeInMonths(Real64 const depth, int const monthOfSim);
+    Real64 getGroundTempAtTimeInMonths(Real64 const depth, int const monthOfSim) override;
```

Comment:
Nice misc cleanups along the way.

Reply:
I'm trying really hard to limit my changes to things that only directly affect the output, but figured I needed to do that when I saw it.